Polygon_mesh_processing/doc/Polygon_mesh_processing/Concepts/PMPHolefillingVisitor.h

```diff
@@ -2,15 +2,14 @@
 /// \cgalConcept
 ///
 /// The concept `PMPHolefillingVisitor` defines the requirements for the visitor
-/// used in \link PMP_hole_filling_grp hole-filling-related functions \endlink to track
-/// the creation of new faces and new edges.
+/// used in \link PMP_hole_filling_grp hole-filling-related functions \endlink.
 /// The hole filling may use a 2D constrained triangulation
-/// for almost planar holes.  If that is not appropriate or fails it
-/// may use an algorithm with at most quadratic running time. If that fails
-/// it uses an algorithm with cubic running time.
+/// for almost planar holes (*planar phase*).  If that is not appropriate or fails it
```

Comment:
Does `*`  become bold?  Bold should be reeserved for headings.

Reply:
italic